doc/xml/Makefile.am

```diff
@@ -44,17 +47,17 @@ all: $(man_MANS) $(HTMLS)
 clean:
 	-test -z "$(CLEAN_FILES)" || rm -f $(CLEAN_FILES)
 
-../man/man1/firewall-cmd.1: errorcodes.xml
+../man/man1/firewall-cmd.1: errorcodes.xml firewall-cmd.xml
 
-../html/firewall-cmd.html: errorcodes.xml
+../html/firewall-cmd.html: errorcodes.xml firewall-cmd.xml
 
-../man/man1/%.1: %.xml authors.xml notes.xml seealso.xml transform-man.xsl
+../man/man1/%.1: %.xml authors.xml notes.xml seealso.xml transform-man.xsl $(XML_FIREWALLD_DEPS)
```

Comment:
Adding XML_FIREWALLD_DEPS here is bogus. The dependency is already there because of the %.xml.
Don't worry about updating this PR. I'll take care of it.